app/src/main/java/org/mozilla/fenix/library/history/HistorySearchDialogFragment.kt

```diff
@@ -254,4 +264,48 @@ class HistorySearchDialogFragment : AppCompatDialogFragment(), UserInteractionHa
             }
         }
     }
+
+    private val startForResult = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+        result: ActivityResult ->
+        if (result.resultCode == Activity.RESULT_OK) {
+            val intent = result.data
+            intent?.getStringArrayListExtra(RecognizerIntent.EXTRA_RESULTS)?.first()?.also {
+                toolbarView.view.edit.updateUrl(url = it, shouldHighlight = true)
+                interactor.onTextChanged(it)
+                toolbarView.view.edit.focus()
+            }
+        }
+    }
```

Comment:
Can you run a code format on this block, I almost expect an extra indent for the if statement block

Reply:
Can't add indent.  Fails ktlint when I add more indent for the if block

Reply:
Sure, it passed ktlint, I'll change the format to your suggestion.